App/Redux/PhotoViewingSelectors.ts

```diff
@@ -161,6 +161,23 @@ export function getThreadThumbs(state: RootState, byAddres: string, sortBy?: 'na
     })
 }
 
+function isDirectMessageThread(address: string): (thread: ThreadData) => boolean {
+  return (thread) => {
+    return thread.whitelist.indexOf(address) !== -1 &&
+           thread.sharing === Thread.Sharing.NOT_SHARED &&
+           thread.type === Thread.Type.OPEN
+  }
+}
+
+export function directMessageThreadExists(state: RootState, address: string) {
```

Comment:
This one seems redundant. The below `getDirectMessageThread` gives us the same information. If it returns `undefined` we know the direct message thread doesn't exist.